### ISSUE TITLE
trezorctl support?

### ISSUE THREAD

**Issue by jonathancross** [brianddk/trezor-tails#3]
The readme doesn't mention if this allows access to the Trezor command line client `trezorctl`. If not, would be nice to add it.

**Comment by brianddk** [comment]
Yes, support is there in most all revisions.  v1.0 folds in support for `trezorctl` v0.12.0.  You should use the `amnesia` account (default) and not `root` or `clearnet`.

I'm adding a wiki and this week and trying to increase the documentation.  Right now it is just stream of consciousnesses on my part.

**Comment by brianddk** [comment]
I've got the [wiki up](https://github.com/brianddk/trezor-tails/wiki), or at least an outline for it.  It shows in the sidebar all the verified products.  `trezorctl` is listed there.  I'll fill in the content over the next few days / weeks hopefully.

**Comment by jonathancross** [comment]
Fantastic!
I'd like to make some small contributions to the Wiki if you are open to that?

**Comment by brianddk** [comment]
Sure... If your familiar with github, I've made a [branch called wiki](https://github.com/brianddk/trezor-tails/wiki) that I try to keep up with the hidden wiki repo.  You can issue a PR to the wiki branch.  Or I can just fork a private repo of the wiki with you as a contributor.

Either way... Yes please... documentation is not my strong suit.

I have been making some changes recently.  I'll be sure to push them in the next few hours.

**Comment by brianddk** [comment]
OK... I've invited you as a contributor to a [wiki contributor's repo](https://github.com/brianddk/trezor-tails-wiki/wiki) and finished my rebase of the wiki commits.  So two ways to contribute to the wiki:

1. Fork [trezor-tails](https://github.com/brianddk/trezor-tails) repo and make a PR to the [wiki branch](https://github.com/brianddk/trezor-tails/tree/wiki).
2. Just make contributions directly to the wiki at [trezor-tails-wiki](https://github.com/brianddk/trezor-tails-wiki/wiki)

Either way, I welcome the efforts.

**Comment by jonathancross** [comment]
@brianddk Seems the invitation has expired, sorry, didn't realize it had a time limit :-/

**Comment by prusnak** [comment]
Tails 4.10 supports Trezor out of the box (again) (see https://github.com/brianddk/trezor-tails/issues/4), so `trezorctl` is available in the default installation by default.
